class-00/lab-b/starter-code/app.js

```diff
@@ -184,17 +184,17 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+// -undefine 

```

Comment:
The 'this' in joe.scope method is not undefined. Since joe.scope executes a console.log you will see a log reported from line 183. Then on line 187 you are console logging the return of the joe.scope function. That return is undefined, but the 'this' from inside joe.scope is defined.